prow/config.yaml

```diff
@@ -2931,7 +2931,7 @@ periodics:
       - name: GOOGLE_APPLICATION_CREDENTIALS
         value: /etc/service-account/service-account.json
       - name: USER
-        value: prow
+        value: jenkins
```

Comment:
the ssh key below is specific for user `prow` though, do you want a new key for `jenkins`?